test/integration/serverTest.js

```diff
@@ -154,6 +154,126 @@ describe('Brave Sensor server', () => {
     await redis.disconnect()
   })
 
+  describe('POST request: alerts from firmware state machine for locations with a non-null Particle Siren Id', () => {
+    beforeEach(async () => {
+      await redis.clearKeys()
+      await db.clearSessions()
+      await db.clearLocations()
+      await db.clearClients()
+      const client = await clientFactory(db)
+      await db.createLocation(
+        testLocation1Id,
+        testLocation1PhoneNumber,
+        MOVEMENT_THRESHOLD,
+        STILLNESS_TIMER,
+        DURATION_TIMER,
+        1000,
+        INITIAL_TIMER,
+        ['+15005550006'],
+        '+15005550006',
+        ['+15005550006'],
+        1000,
+        'locationName',
+        door_coreID,
+        radar_coreID,
+        'XeThru',
+        'alertApiKey',
+        true,
+        true,
+        'particleCoreIdTest',
+        client.id,
```

Comment:
if you change the order of parameters above to match the query string, you should also change this to 
```
        client.id,
        'particleCoreIdTest',
```

Reply:
Did not change the order of parameters - if the query string is different it is the what should be changed. I don't know where the query string is - could you point me to it?

Reply:
my mistake! disregard